src/current/cockroachcloud/create-an-advanced-cluster.md

```diff
@@ -0,0 +1,186 @@
+
+    Factor | Description
+    ----------|------------
+    Capacity | Total raw data size you expect to store without replication.
+    Replication | The default replication factor for a CockroachDB {{ site.data.products.cloud }} cluster is 3.
+    Buffer | Additional buffer (overhead data, accounting for data growth, etc.). If you are importing an existing dataset, we recommend you provision at least 50% additional storage to account for the import functionality.
+    Compression | The percentage of savings you can expect to achieve with compression. With CockroachDB's default compression algorithm, we typically see about a 40% savings on raw data size.
+
+    For more details about disk performance on a given cloud provider, refer to: <ul><li><b>GCP</b>: <a href="https://cloud.google.com/compute/docs/disks/performance">Configure disks to meet performance requirements</a></li><li><b>AWS</b>: <a href="https://aws.amazon.com/ebs/features/#Amazon_EBS_volume_types">Amazon EBS volume types</a></li><li><b>Azure</b>: <a href="https://learn.microsoft.com/azure/virtual-machines/disks-performance">Virtual machine and disk performance</a></li></ul>
+
+To change the hardware configuration after the cluster is created, see [Manage a CockroachDB {{ site.data.products.advanced }} Cluster]({% link cockroachcloud/cluster-management.md %}).
+
+See the [Example]({% link cockroachcloud/plan-your-cluster.md %}#example) for further guidance.
+
+
+Click **Next: Security**.
+
+## Step 6. Configure advanced security features
+
+You can add advanced security features for PCI and HIPAA [compliance]({% link cockroachcloud/compliance.md %}) at an additional cost. 
+
+    {{site.data.alerts.callout_danger}}
+    This configuration cannot be changed after cluster creation.
+    {{site.data.alerts.end}}
+
+## Step 7. Enter billing details
+
+1. On the **Finalize** page, verify your selections for the cloud provider, region(s), number of nodes, and the capacity.
+
+        Once your cluster is created, see [Establish VPC Peering or AWS PrivateLink]({% link cockroachcloud/connect-to-your-cluster.md %}#establish-gcp-vpc-peering-or-aws-privatelink) to finish setting up VPC Peering for your cluster.
+
+1. Verify the hourly estimated cost for the cluster. The cost displayed does not include taxes.
+
+    You will be billed monthly.
+
+1. Add your preferred [payment method]({% link cockroachcloud/billing-management.md %}).
+1. [If applicable]({% link cockroachcloud/frequently-asked-questions.md %}#how-do-cockroachdb-free-trials-work), the 30-day trial code is pre-applied to your cluster.
+      {{site.data.alerts.callout_info}}
+      Make sure that you [delete your trial cluster]({% link cockroachcloud/cluster-management.md %}#delete-cluster) before the trial expires. Your credit card will be charged after the trial ends. You can check the validity of the code on the [Billing]({% link cockroachcloud/billing-management.md %}) page.
```

Comment:
```suggestion
      Remember to [delete your trial cluster]({% link cockroachcloud/cluster-management.md %}#delete-cluster) before the trial expires. Otherwise, your credit card will be charged after the trial ends. You can check the validity of the code on the [Billing]({% link cockroachcloud/billing-management.md %}) page.
```